src/components/Sidebar.tsx

```diff
@@ -2,57 +2,59 @@ import Link from "next/link";
 import Image from "next/legacy/image";
 import { signOut } from "next-auth/react";
 
-const SidebarItem = ({ label } : { label: string }) => {
+const SidebarItem = ({ label }: { label: string }) => {
   return (
-    <div className="w-full px-4 p-2 text-md text-left font-serif duration-150 hover:bg-darker-tan">
+    <div className="text-md w-full p-2 px-4 text-left font-serif duration-150 hover:bg-darker-tan">
       <div className="duration-150 hover:translate-x-1">{label}</div>
     </div>
   );
 };
 
-const Sidebar = ({ displayName } : { displayName: string }) => {
+const Sidebar = ({ displayName }: { displayName: string }) => {
   return (
-    <div className="flex flex-col w-[160px] h-screen top-0 py-12 justify-start gap-20 \
-                    bg-med-tan border-r border-darker-tan">
-          <div className="flex flex-col p-4 place-items-center">
-            <Link href="/">
-              <h1 className="font-serif text-center text-2xl duration-150 hover:-translate-y-0.5 z-10">
-                The Legacy Project
-              </h1>
-            </Link>
+    <div
+      className="\ sticky top-0 flex h-screen w-[160px] flex-col justify-start gap-20
```

Comment:
This is a good addition to have. I would revert the `sticky` style since it may conflict with the team that's working on Navigation Bar.